stable_baselines3/ppo/ppo.py

```diff
@@ -331,21 +354,19 @@ def learn(self,
             # Display training infos
             if self.verbose >= 1 and log_interval is not None and iteration % log_interval == 0:
                 fps = int(self.num_timesteps / (time.time() - self.start_time))
-                logger.logkv("iterations", iteration)
+                logger.logkv("iterations", iteration, exclude='tensorboard')
                 if len(self.ep_info_buffer) > 0 and len(self.ep_info_buffer[0]) > 0:
-                    logger.logkv('ep_rew_mean', self.safe_mean([ep_info['r'] for ep_info in self.ep_info_buffer]))
-                    logger.logkv('ep_len_mean', self.safe_mean([ep_info['l'] for ep_info in self.ep_info_buffer]))
-                logger.logkv("fps", fps)
-                logger.logkv('time_elapsed', int(time.time() - self.start_time))
-                logger.logkv("total timesteps", self.num_timesteps)
-                logger.dumpkvs()
+                    logger.logkv('ep_rew_mean', self.safe_mean([ep_info['r']
+                                                                for ep_info in self.ep_info_buffer]), exclude='tensorboard')
```

Comment:
why do you exclude that one?
this is one of the most helpful metrics....

Reply:
Visualizing Iterations in tensorboard? Isn't it just a linear plot?

Reply:
`ep_rew_mean` is the mean episode reward

Reply:
Ah, by mistake I guess